logger/tests/integration_tests.rs

```diff
@@ -323,64 +269,47 @@ async fn generate_and_stream_logs() {
     assert_eq!(
         MinLogItem::from(log),
         MinLogItem {
-            level: LogLevel::Info,
-            fields: json!({"message": "[span] span_name2", "deployment_id": quoted_deployment_id}),
+            data: format!("TRACE span_name2{{deployment_id=\"{DEPLOYMENT_ID}\"}}: integration_tests: inside span 2 event\n")
         },
     );
 }
 
 /// For the service logs the deployment id will be retrieved from the spans of functions
 /// instrumented with the deployment_id field, this way we can choose which spans we want
 /// to associate with a deployment and record in the logger.
-fn generate_service_logs(port: u16, deployment_id: String, generator: fn(String)) {
-    generate_logs(
-        port,
-        deployment_id,
-        generator,
-        vec![KeyValue::new("service.name", "test")],
-    );
+fn generate_service_logs(
```

Comment:
It looks like this and generate_runtime_logs are duplicated now

Reply:
Yep, fixed this by removing the tracing layer, and testing just the RPCs.